### PR TITLE
check in manifests for each env

### PR DESCRIPTION
**Pull request by afomi**
https://github.com/GSA/touchpoints/issues/52

### CODE REVIEW

File: touchpoints-demo.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      S3_AWS_BUCKET_NAME:
+      S3_AWS_HOST:
+      S3_AWS_REGION:
+      S3_AWS_SECRET_ACCESS_KEY:
+      TOUCHPOINTS_EMAIL_SENDER:
+      TOUCHPOINTS_GTM_CONTAINER_ID:
+      TOUCHPOINTS_WEB_DOMAIN: touchpoints-demo.app.cloud.gov
+    buildpacks:
+      - https://github.com/cloudfoundry/nodejs-buildpack.git
+      - https://github.com/cloudfoundry/ruby-buildpack.git
```
Comment (afeld) [inline touchpoints-demo.yml]: I recommend using the cloud.gov-provided ones if you can: `nodejs_buildpack` and `ruby_buildpack`. They have more compliance baked in than the public ones.

File: touchpoints-demo.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      LOGIN_GOV_REDIRECT_URI: https://touchpoints-demo.app.cloud.gov/users/auth/login_dot_gov/callback
+      NEW_RELIC_KEY:
+      RAILS_ENV: production
+      S3_AWS_ACCESS_KEY_ID:
+      S3_AWS_BUCKET_NAME:
+      S3_AWS_HOST:
+      S3_AWS_REGION:
+      S3_AWS_SECRET_ACCESS_KEY:
+      TOUCHPOINTS_EMAIL_SENDER:
+      TOUCHPOINTS_GTM_CONTAINER_ID:
```
Comment (afeld) [inline touchpoints-demo.yml]: You could probably leave out the environment variables whose values aren't in this file, if you want. Fine either way.

Comment (afomi) [reply touchpoints-demo.yml]: will leave them in for now. i might reference the manifest.sample later with documentation to ensure a new dev picks these up.   leaving them here sorta prompts the config.

thanks!

File: touchpoints-demo.yml (hunk 1 above)
Comment (afomi) [reply touchpoints-demo.yml]: nice.  will do. thanks!

**Pull request merged**